### PR TITLE
feat: add marketing widget to all snapshot spaces that use osnap

### PR DESCRIPTION
**Pull request by gsteenkamp89**
Feature #UMA-1663

screenshots

in CREATE proposal

Screenshot 2023-11-28 at 19 26 46

Handle Outcome

Screenshot 2023-11-28 at 19 27 07

### CODE REVIEW

File: src/plugins/oSnap/Proposal.vue [hunk 1]
```diff
@@ -94,8 +94,9 @@ function enrichTransactionForDisplay(transaction: Transaction) {
 <template>
   <template v-if="safe.transactions.length > 0">
     <div
-      class="flex w-full flex-col gap-4 rounded-2xl border border-gray-200 p-3 md:p-4"
+      class="flex w-full flex-col gap-4 rounded-2xl border border-gray-200 p-3 md:p-4 relative"
     >
+      <OsnapMarketingWidget class="absolute top-[-16px] right-[16px]" />
```
Comment (daywiss) [inline src/plugins/oSnap/Proposal.vue:99]: i think we might need logic to detect if the transactions are based on osnap

**Review by daywiss — COMMENTED** [review]
not seeing this show up for some reason

File: src/plugins/oSnap/Proposal.vue (hunk 1 above)
Comment (gsteenkamp89) [reply src/plugins/oSnap/Proposal.vue:100]: this lives inside our plugin specific osnap proposal component so need for that.

**Comment by daywiss** [comment]
something is really wrong here, its showing that all proposals dont exist "can't find any proposals". maybe we need to get latest changes in from snapshot

**Review by daywiss — APPROVED** [review]
got it workign after rebase. looks great.  make sure you pr this to snapshot repo, you dont need to merge it to our master

**Pull request closed without merging**